src/Controllers/MyRadio/removeActionPermission.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Provides a tool to remove action permissions for MyRadio Service/Module/Action systems.
+ */
+use \MyRadio\MyRadio\AuthUtils;
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+/*
+ * Form definition for removing permission.
+ */
+$form = new MyRadioForm(
+    'add_permission',
+    $module,
+    $action,
+    [
+        'title' => 'Permissions',
+        'subtitle' => 'Delete Action Permission',
+    ]
+)->addField(
+    new MyRadioFormField(
+        'permissionid',
+        MyRadioFormField::TYPE_HIDDEN,
+        [
+            'required' => true
+        ]
+    )
+)->addField(
+    new MyRadioFormField(
+        'module',
+        MyRadioFormField::TYPE_TEXT,
+        [
+            'enabled' => false
+        ]
+    )
+)->addField(
+    new MyRadioFormField(
+        'action',
+        MyRadioFormField::TYPE_TEXT,
+        [
+            'enabled' => false
+        ]
+    )
+)->addField(
+    new MyRadioFormField(
+        'permission',
+        MyRadioFormField::TYPE_TEXT,
+        [
+            'enabled' => false
+        ]
+    )
+)->addField(
+    new MyRadioFormField(
+        'confirm',
+        MyRadioFormField::TYPE_CHECK,
+        [
+            'explanation' => '<strong>I confirm that deleting this action permission is
```

Comment:
Put the name of the permission in the message?

Reply:
<img width="962" alt="screen shot 2018-04-01 at 17 53 03" src="https://user-images.githubusercontent.com/1026879/38175528-9a0bafa6-35d5-11e8-88da-bff4f150f1d9.png">
It is already above in the form, I deemed that good enough?